Makefile.toml

```diff
@@ -91,7 +91,17 @@ end
 description = "Builds all test components with golem-cli"
 script_runner = "@duckscript"
 script = '''
-domains = array llm
+domains = array llm web-search
+
+# since it depends on build artifacts
+if is_empty ${1}
```

Comment:
you can remove this the build is done internally in the makefile of the relevant domain